README.md

```diff
@@ -121,12 +120,35 @@ following JSON specification:
 <a id="markdown-quick-start" name="quick-start"></a>
 ## Quick Start
 
-<Hello world style example.>
+### Pre-requisite
+
+* Install docker and docker-compose.
+* Read up on [serverfull](https://github.com/asecurityteam/serverfull)
```

Comment:
Is this an actual prereq to the quick start? This seems like something that could be learned _after_ doing a quick run locally.

Reply:
Good point